docs/cli-reference.md

```
@@ -367,6 +367,70 @@ cloud subnet rm my-private-subnet --vpc my-network
+```
+
+### `sg add-rule <sg-id>`
+
+Add a rule to a security group.
+
+```bash
+cloud sg add-rule my-sg --protocol tcp --port-min 80 --port-max 80 --cidr 0.0.0.0/0
+```
+
+**Flags**:
+| Flag | Default | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)